src/VirtoCommerce.ElasticSearchModule.Data/VirtoCommerce.ElasticSearchModule.Data.csproj

```diff
@@ -5,9 +5,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Elasticsearch.Net" Version="5.6.5" />
-    <PackageReference Include="NEST" Version="5.6.5" />
-      <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+    <PackageReference Include="Elasticsearch.Net" Version="7.4.2" />
```

Comment:
did you test after update? and checked Integration Tests?

Reply:
Yes. I did.